src/Signal.elm

```diff
@@ -148,7 +149,9 @@ update wins, just like with `merge`.
 -}
 mergeMany : List (Signal a) -> Signal a
 mergeMany signals =
-    List.foldr1 merge signals
+    case signals of
+        s :: rest -> List.foldr merge s rest
```

Comment:
I think this is buggy. `s` is the leftmost thing, but it is used as the rightmost. I think `foldr` needs the last element.